src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
The method `getStatistic` is public. According to the checklist, if the method has only a utility purpose and is used only inside the same class, it should not be `public`. Keep your code as close as possible to follow the encapsulation principle.